### PR TITLE
FE 101 - Detail Page

### PR DESCRIPTION
**Pull request by lpzchrstn**
# Related Tickets
[FE-430](https://codeandtheory.atlassian.net/browse/FE-430)

# Description
This PR has the default page structure and design of the Detail Page.

# Validation
No Validation yet for behavior, the page just needs to look like this site: https://www.christopher-ware.com/vsa

Here are some images related to this PR:
Screen Shot 2022-01-11 at 3 31 50 PM
Screen Shot 2022-01-11 at 3 32 24 PM
Screen Shot 2022-01-11 at 3 33 19 PM

# Risks
This PR is using the compiled scripts and styles and can be generated by running  `npm start build`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    "cleanup-js": "rm -rf ./dist/js",
+    "mkdirs": "mkdir -p ./dist/js && mkdir -p ./dist/css",
+    "compile-assets": "npm run compile-css && npm run compile-js",
+    "compile-assets-prod": "npm run compile-css-prod && npm run compile-js-prod",
+    "compile-css": "sass ./src/sass/main.scss:./dist/css/main.css",
+    "compile-css-prod": "sass --no-source-map --style=compressed ./src/sass/main.scss:./dist/css/main.css",
+    "watching-css": "sass --watch ./src/sass/main.scss:./dist/css/main.css",
+    "compile-js": "rollup -c ./src/config/rollup.config.js",
+    "compile-js-prod": "rollup -c ./src/config/rollup.config-prod.js",
+    "watching-js": "rollup -c ./src/config/rollup.config.js -w"
```
Comment (joshuacerbito) [inline package.json:22]: Please review the comment from https://github.com/lpzchrstn/sass-lpzchrstn/pull/1 as it's very much the same as this.

File: .gitignore [hunk 2]
```diff
@@ -1,4 +1,5 @@
 #Insert ignored files here
 node_modules/
 main.css
-main.css.map
\ No newline at end of file
+main.css.map
+dist/
```
Comment (joshuacerbito) [inline .gitignore:5]: Please make sure that there's always an empty line at the end of each file

File: src/sass/main.scss [hunk 3]
```diff
@@ -0,0 +1,4 @@
+//Core
+@import 'modules/keyframes.scss';
+@import 'modules/core.scss';
+@import 'modules/fonts.scss';
```
Comment (joshuacerbito) [inline src/sass/main.scss:4]: Please make sure that there's always an empty line at the end of each file

File: src/sass/modules/fonts.scss [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    font-family: "Tiempos Headline";
+    src: url("../../src/fonts/TiemposHeadline/TiemposHeadline-Bold.otf");
+    font-weight: 800;
+}
+
+@font-face {
+    font-family: "Tiempos Headline";
+    src: url("../../src/fonts/TiemposHeadline/TiemposHeadline-Black.otf");
+    font-weight: 900;
+}
```
Comment (joshuacerbito) [inline src/sass/modules/fonts.scss:35]: Please make sure that there's always an empty line at the end of each file

File: src/sass/modules/keyframes.scss [hunk 5]
```diff
@@ -0,0 +1,8 @@
+@keyframes arrow-btn-pulse {
+  0% { 
+    box-shadow: 0 0 0 0 rgb(255 255 255 / 10%);
+  }
+  100% {
+    box-shadow: 0 0 0 20px rgb(255 255 255 / 0%);
+  }
+}
```
Comment (joshuacerbito) [inline src/sass/modules/keyframes.scss:8]: Please make sure that there's always an empty line at the end of each file

**Review by joshuacerbito — CHANGES_REQUESTED** [review]
Left a few comments for you to address and consider, @lpzchrstn

File: package.json (hunk 1 above)
Comment (lpzchrstn) [reply package.json]: Oh right, I just copied the package.json from that PR but forgot to update it.

**Review by joshuacerbito — APPROVED** [review]
All of these look good. For the next PRs, please make sure to always close the loop on the conversations.

**Pull request merged**